package.json

```diff
@@ -2,7 +2,7 @@
   "name": "3box",
   "version": "1.1.0",
   "description": "Interact with user data",
-  "main": "lib/3box.js",
+  "main": "dist/3box.js",
```

Comment:
The js fine in the dist directory is a file compiled with all the dependencies into one single file. Using it as the main file is not so good for people that have their own build processes set up to optimize all dependencies since 3box will only be a blob. the `lib` folder contains just the 3box code (compiled down to ES5) without the dependencies baked in.